Vc/avx/vector.h

```diff
@@ -159,6 +159,8 @@ template <typename T> class Vector<T, VectorAbi::Avx>
             explicit Vector(U &&x) Vc_INTRINSIC_R;
 #endif
 
+        Vc_INTRINSIC Vector(reference a) : Vector(static_cast<EntryType>(a)) {}
+
         ///////////////////////////////////////////////////////////////////////////////////////////
         // broadcast
         Vc_INTRINSIC Vector(EntryType a) : d(Detail::avx_broadcast(a)) {}
```

Comment:
From my gut feeling, I would have made this ctor `explicit`. But then `Vector(EntryType)` is also not explicit. So I guess I am fine with your solution.

Reply:
It seems your gut is right, not making it explicit introduces ambiguities across all constructors.